### PR TITLE
mini update 9: describe content management workflow

### PR DESCRIPTION
**Pull request by pmengelbert**
Builds upon PR #168 .

The relevant diff can be found here: 
https://github.com/bloodorangeio/distribution-spec/compare/mini-update-8...bloodorangeio:mini-update-9?expand=1

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -79,19 +79,283 @@ Registry providers can self-certify by submitting conformance results to [openco
...
+`<name>` is the namespace of the repository, `<int>` is the number of tags requested, and `<tagname>` is the *value* of
+the last tag. `<tagname>` MUST NOT be an index, but rather it MUST be a proper tag. A request of this sort will return
+up to `<int>` tags, beginning non-inclusively with `<tagname>`. That is to say, `<tagname>` will not be included in the
+results, but up to `<int>` tags *after* `<tagname>` will be returned. The tags MUST be in lexical order.
 
 #### Content Management
 
-TODO: describe the Content Management category and the high-level details
+##### Deleting tags
+Content management refers to the deletion of blobs, tags and manifests. Registries MAY implement deletion or they MAY dis-
```
Comment (jdolitsky) [inline spec.md:336]: nit: oxford comma

File: spec.md [hunk 2]
```diff
@@ -79,19 +79,283 @@ Registry providers can self-certify by submitting conformance results to [openco
...
+able it. Similarly, a registry MAY implement tag deletion, while others MAY allow deletion only by manifest.
```
Comment (jdolitsky) [inline spec.md:337]: disable has a hyphen and a newline

File: spec.md [hunk 3]
```diff
@@ -79,19 +79,283 @@ Registry providers can self-certify by submitting conformance results to [openco
...
+
+`<name>` is the namespace of the repository, and `<tag>` is the name of the tag to be deleted. Upon success, the registry
+MUST respond with a `202 Accepted` code. If tag deletion is disabled, the registry MUST respond with a `400 Bad Request`
```
Comment (jdolitsky) [inline spec.md:340]: Should also allow 405. See PR: https://github.com/opencontainers/distribution-spec/pull/171

File: spec.md (hunk 1 above)
Comment (jdolitsky) [inline spec.md:336]: This whole section is not specific to deleting tags, should maybe be removed or moved to above

File: spec.md [hunk 4]
```diff
@@ -79,19 +79,283 @@ Registry providers can self-certify by submitting conformance results to [openco
...
+
+To delete a tag, perform a `DELETE` request to a path in the following format:
+`/v2/<name>/manifests/<tag>`
+
+##### Deleting Manifests
+To delete a manifest, perform a `DELETE` request to a path in the following format:
+`/v2/<name>/manifests/<digest>`
+
+`<name>` is the namespace of the repository, and `<digest>` is the digest of the manifest to be deleted. Upon success, the registry
+MUST respond with a `202 Accepted` code. If the repository does not exist, the response MUST bear the code `404 Not Found`.
```
Comment (jdolitsky) [inline spec.md:351]: "response MUST return `404 Not Found`"

Comment (pmengelbert) [reply spec.md:351]: The response must bear the code deep into Mordor

**Comment by jdolitsky** [comment]
Closing this in favor of https://github.com/opencontainers/distribution-spec/pull/178

**Pull request closed without merging**